src/core/stdc/stdio.d

```diff
@@ -391,7 +391,21 @@ else version (CRuntime_Microsoft)
     ///
     struct _iobuf
     {
-        void* undefined;
+        version (MinGW)
+        {
+            char* _ptr;
+            int _cnt;
+            char* _base;
+            int _flag;
+            int _file;
+            int _charbuf;
+            int _bufsiz;
+            char* _tmpfname;
```

Comment:
Does MinGW really have its own `_iobuf`, not inheriting the MSVC one? - It's one of the structs which has seen a major change with the introduction of the UCRT with MSVC 2015 IIRC.

Reply:
I don't see it in my locally installed msys2 environment, but it appears upstream mingw have gotten round to adding in UCRT `_iobuf`.
https://github.com/mirror/mingw-w64/blob/1c773877f4a13c8bd7bfb8da80e1e8761a889f51/mingw-w64-headers/crt/stdio.h#L24-L37

However, given your previous comment, I guess I could instead put the `_fputc_unlocked` definitions inside an MSVC support module.

Reply:
I've implemented `_fgetc_unlocked` and `_fputc_unlocked` in an msvc support module, these definitions are no longer required to be exported.